edb/pgsql/metaschema.py

```diff
@@ -6872,9 +6877,9 @@ def _generate_sql_information_schema(
           TRUE AS connoinherit,
           ARRAY[pa.attnum]::int2[] AS conkey,
           ARRAY[1]::int2[] AS confkey, -- id will always have attnum 1
-          ARRAY[2972]::oid[] AS conpfeqop, -- 2972 is eq comparison for uuids
-          ARRAY[2972]::oid[] AS conppeqop, -- 2972 is eq comparison for uuids
-          ARRAY[2972]::oid[] AS conffeqop, -- 2972 is eq comparison for uuids
+          ARRAY['uuid_eq'::regproc]::oid[] AS conpfeqop,
+          ARRAY['uuid_eq'::regproc]::oid[] AS conppeqop,
+          ARRAY['uuid_eq'::regproc]::oid[] AS conffeqop,
```

Comment:
I did check that this id has not changed since pg9. When I was querying pg_catalog, ive realized i can just cast into regclass and make it much more reliable and readable.